docs/source/en/model_doc/rt_detr.md

```diff
@@ -82,4 +93,4 @@ results = image_processor.post_process_object_detection(outputs, target_sizes=to
 ## RTDetrResNetBackbone
 
 [[autodoc]] RTDetrResNetBackbone
-    - forward
\ No newline at end of file
+    - forward
```

Comment:
is this probably a bug with diff? the line exists in the original file

https://github.com/huggingface/transformers/blob/main/docs/source/en/model_doc/rt_detr.md